transport_plugins/bled112/iotile_transport_bled112/bled112.py

```diff
@@ -800,7 +901,9 @@ def _on_disconnect(self, result):
         connection_id = context['connection_id']
         handle = context['handle']
 
-        callback(connection_id, self.id, success, "No reason given")
+        # TODO why self.id is used here, connection handle should be an argument?
```

Comment:
I would prefer this TODO to be an open issue rather than buried in committed source code.

Reply:
I see your point, 
I added this TODO to draw attention during review, I am not sure if I should have done it in another way